### DISCUSSION TITLE
Build problems. Missing gmock.h

### DISCUSSION THREAD

**Discussion by oharboe** [The-OpenROAD-Project/OpenROAD#3513] [General]
There has been a lot of activity concerning build errors, but I tried with latest master merged with #3512

Is it enough to update to latest master or do I need to purge something or other locally?

```
./build_openroad.sh --local --openroad-args "-DCMAKE_BUILD_TYPE=RELEASE -DCMAKE_CXX_FLAGS=-g"
```

```
[ 25%] Building CXX object src/odb/test/cpp/CMakeFiles/TestGuide.dir/TestGuide.cpp.o
/home/oyvind/OpenROAD-flow-scripts/tools/OpenROAD/src/odb/test/cpp/TestAbstractLef.cc:13:10: fatal error: gmock/gmock.h: No such file or directory
   13 | #include "gmock/gmock.h"
      |          ^~~~~~~~~~~~~~~
compilation terminated.
gmake[2]: *** [src/odb/test/cpp/CMakeFiles/OdbGTests.dir/build.make:90: src/odb/test/cpp/CMakeFiles/OdbGTests.dir/TestAbstractLef.cc.o] Error 1
gmake[1]: *** [CMakeFiles/Makefile2:2331: src/odb/test/cpp/CMakeFiles/OdbGTests.dir/all] Error 2
gmake[1]: *** Waiting for unfinished jobs...
```

**Comment by QuantamHD** [comment]
This ones my bad. It seems that some installations of libgtest-dev don't include google mock. If you uninstall your system gtest it will fix this error, but I'll see if there's something else we could do to fix this.

**Reply by QuantamHD** [reply]
Or the other thing you could try is to remove this line. https://github.com/The-OpenROAD-Project/OpenROAD/blob/master/third-party/gtest/CMakeLists.txt#L1

**Comment by vvbandeira** [comment]
Fixed on #3514
